pom.xml

```diff
@@ -246,12 +247,12 @@
 			<dependency>
 				<groupId>org.eclipse.jetty</groupId>
 				<artifactId>jetty-servlet</artifactId>
-				<version>${jetty.version}</version>
+				<version>11.0.24</version>
```

Comment:
There is an explicit version here, but `jetty.version` is set above and used below. Maybe comment why it has to be this specific one?